### ISSUE TITLE
TDB parser silently fails on commands without all uppercase characters

### ISSUE THREAD

**Issue by richardotis** [sundmanbo/opencalphad#14]
Reproducible Example
-------
```
 Phase TEST % 1 1 !
 Constituent TEST : H,HE,LI,BE,B,C,N,O,F,NE : !
```

Observed Behavior
-------
"list data" will not show the phase or any of its parameters.
No calculations will work and OC will complain about missing data.

Expected Behavior
--------
The TDB parser should not care about capitalization of the commands, or it should at least warn the user.

Workaround
--------------
Modify the TDB file to put the commands in all uppercase.

**Comment by sundmanbo** [comment]
I notice also keywords had to be upper case.  Fixed but not uploaded yet.

/Bosse


2015-02-28 1:48 GMT+01:00 Richard Otis <notifications@github.com>:

**Comment by richardotis** [comment]
When you commit your fix, if you put something like "fixes #14" in the commit message, GitHub will automatically mark the issue as closed and tag the issue with the fix. It'll also notify everyone who posted in the thread that you fixed it.
